src/VisualStudio/VisualBasic/Impl/CodeModel/VisualBasicCodeModelService.vb

```diff
@@ -800,7 +800,7 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.CodeModel
         End Function
 
         Public Overrides Function GetUnescapedName(name As String) As String
-            Return If(name IsNot Nothing AndAlso name.Length > 2 AndAlso name.StartsWith("[") AndAlso name.EndsWith("]"),
+            Return If(name IsNot Nothing AndAlso name.Length > 2 AndAlso name.StartsWith("[", StringComparison.Ordinal) AndAlso name.EndsWith("]", StringComparison.Ordinal),
```

Comment:
How about adding an optimized helper (extension?) method for IsEscapedName? I'm sure I've come across this ```StartsWith("["  ... AndAlso EndsWith("]")``` expression elsewhere.

    Public Shared Function IsEscapedName(name As String) As Boolean
        Return name IsNot Nothing AndAlso name.Length > 2 AndAlso name[0] == '[' AndAlso name[name.Length-1] == ']'
    End Function